framework/h5py_interface_creator.py

```diff
@@ -186,7 +186,12 @@ def addGroup(self,rlz):
       @ Out, None
     """
     parentID  = rlz.get("RAVEN_parentID",[None])[0]
-    groupName = str(rlz.get("prefix")[0] if not isinstance(rlz.get("prefix"),basestring) else rlz.get("prefix"))
+    prefix    = rlz.get("prefix")
```

Comment:
The issue in #780 complains that ```rlz``` is None, the changes here did not reflect this. Could you explain it?

Reply:
Sure. The issue reports a failure in the following line:
```
groupName = str(rlz.get("prefix")[0] if not isinstance(rlz.get("prefix"),basestring) else rlz.get("prefix"))
```
The reason was that since some of the adaptive samplers/optimizers construct a solution export DataObject without a prefix for each realization, the ```rlz.get("prefix")``` was returning None (not found keyword "prefix"). Now since the old machine was either expecting the prefix to be a string (```basestring```) or a list, it was falling in the "not string case" so trying to access to ```rlz.get("prefix")[0]``` but the ruturn case was not a string not because it was a list but because it was None.

Reply:
Got it. Thanks.